stock_picking_partner_block_out/models/stock_picking.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2024 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class StockPicking(models.Model):
+
+    _inherit = "stock.picking"
+
+    partner_picking_block_out = fields.Boolean(
```

Comment:
@AnizR A value change on the partner side will recompute all the values for all the pickings. Do you really want to store it ?